src/services/TemplatesService.js

```diff
@@ -0,0 +1,26 @@
+import { api } from 'boot/axios';
+import { getTemplatesByType } from 'src/composables/TemplateManager';
+
+/**
+ * @param {string} type - project | component | model
+ * @returns {Promise<object>}
+ * otherwise an error.
+ */
+export async function getAPITemplatesByType(type) {
+  return api.get(`/libraries/templates?type=${type.toUpperCase().trim()}`);
+}
+
+/**
+ * Get templates have "project" type.
+ * @returns {Array} return array of project templates.
+ */
+export async function getProjectTemplates() {
+  if (!process.env.HAS_BACKEND) {
+    const response = await getTemplatesByType('project');
+    return response;
```

Comment:
why do you have a `const response`, you can directly 
`return getTemplatesByType(...)`

Reply:
Done